src/helpers/index.ts

```diff
@@ -140,3 +140,18 @@ export async function installPackage(dependency: DependencyResolved): Promise<vo
   error.stack = stderr.trim()
   throw error
 }
+
+
+export async function getResolvedDependency(item: string | Dependency): Promise<DependencyResolved> {
+  // string entry
+  if (typeof item === 'string') {
+    return { name: item, directory: await resolveDependencyPath(item) }
+  }
+
+  if ('name' in item) {
+    return { ...item, directory: await resolveDependencyPath(item.name) }
+  }
+
+  console.error(`The package-deps entry ${JSON.stringify(item)} is not valid. Please see https://github.com/steelbrain/package-deps#how-it-works`)
```

Comment:
```suggestion
  console.error(`This package-deps entry is not valid. Please see https://github.com/steelbrain/package-deps#how-it-works`, {entry: item})
```

Reply:
Sorry for being nitpicky, last thing and then we're good to go

Reply:
No worries